requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest
```

Comment:
I would rather not have this and instead added an error message with `pip install perm` to benchmarks.py, because this list is not exhaustive -- one of the examples requires Python3 and aiohttp.